types/src/jsonrpc/wrapped/batches.rs

```diff
@@ -97,7 +93,7 @@ pub struct BatchesElem<'a, T> {
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub struct BatchesElemId {
 	/// Id of the batch within `BatchesState::batches`.
-	outer: u64,
+	outer: usize,
```

Comment:
I assume this isn't going to cause any problems in Wasm since the type is not serializable, and the other member is `usize` already.